front/src/index.js

```diff
@@ -1,10 +1,23 @@
-import React from "react";
+import React, { useState, useEffect} from "react";
 import ReactDOM from "react-dom";
 // import './styles/style.css'
 import './styles/style.scss'
 
 const App = () => {
-    return <h1>Hello from react and sass</h1>
+    let [description, setDescription] = useState([]);
+
+    useEffect(() => {
+        getNotes()
+    })
```

Comment:
```suggestion
    }, [setDescription, getNotes])
```
https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect